achecker.js

```diff
@@ -10,7 +10,7 @@
 const path = require('path');
 
 module.exports = {
-  ruleArchive: 'latest',
+  ruleArchive: 'versioned',
```

Comment:
This makes sure we use a rule pack that matches the version of `accessibility-checker` we're currently on. New rulesets will only be brought in by updating `accessibility-checker`, so no more randomly failing CI checks out of the blue after a new ruleset is published on `latest`.